standard/statements.md

```diff
@@ -327,6 +335,7 @@ In the context of a local variable declaration, the identifier `var` acts as a c
 
 > *Example*: The following are incorrect implicitly typed local variable declarations:
 >
+> <!-- Example: {template:"standalone-console", name:"LocalVariableDecls1", expectedErrors:["CS0818","CS0820","CS0815","CS8917","CS0841"], expectedWarnings:["CS0168"]} -->
```

Comment:
Ignore the warning? (Once everything is in, we should probably perform an audit that all the expected warnings correspond to descriptions in the text.)

Reply:
agreed